### DISCUSSION TITLE
How to add a serverpod project to version control ?

### DISCUSSION THREAD

**Discussion by mozexty** [serverpod/serverpod#409] [Q&A] [answered]
Serverpod produces 3 projects in a main folder.
Is it better to add each project to version control separately ? Or add the main project instead ?

**Comment by yahu1031** [comment, marked as answer]
@mozexty Add all the 3 in a single repository itself if you are not pretty good with version control. It will be easy to handle in deployment too. If you are super good with VC, Then you can also go with server+client in one and flutter one. But need to write the deployment script separately for that and utilizing the client code package will be a little different. I recommend going with all 3 in one repo.

**Comment by mozexty** [comment]
@yahu1031 Thank you

**Comment by yahu1031** [comment]
@mozexty Try making this as discussions 😉👍

**Comment by yahu1031** [comment]
@mozexty Try making these as discussions next time😉👍

**Comment by mozexty** [comment]
@yahu1031 Got it.
